packages/backend/routes/v1/chat/auth.ts

```diff
@@ -16,6 +16,19 @@ const authRouter = express.Router();
 /**
  * OAuth API
  */
+
+// Below route is a quick test endpoint as client package was not working in my case
+authRouter.get('/discord-login', async (_, res) => {
+
+
+
+    // Replace 'YOUR_BOT_TOKEN' with your bot's token
+
+  const discordButton = `<a href="https://discord.com/api/oauth2/authorize?client_id=1163776179002683402&permissions=8&redirect_uri=http%3A%2F%2Flocalhost%3A4001%2Fauth%2Fdiscord%2Fcallback&response_type=code&scope=identify%20messages.read%20applications.commands%20bot" /></a>`;
```

Comment:
Since this piece of code seems to only help locally. Kindly remove this from the PR.